diagnostic.rb

```diff
@@ -10,7 +12,12 @@
 end
 
 # question 4:
+Licensee.where hair_color: %w(Hazel), eye_color: %(Black Brown)
 
 # question 5:
+Licensee.where surname: %w(Escobar Whaley) given_name: %w(Marilynn Chris)
```

Comment:
You're also not storing the results from the query anywhere... which would be fine if you were just operating on the results directly, but you aren't.